apps/trading/components/web3-container/web3-container.spec.tsx

```diff
@@ -17,13 +17,15 @@ jest.mock('@web3-react/core', () => {
   };
 });
 
-test('Prompt to connect opens dialog', () => {
+test('Prompt to connect opens dialog', async () => {
   mockHookValue = defaultHookValue;
-  render(
-    <Web3Container>
-      <div>Child</div>
-    </Web3Container>
-  );
+  await act(async () => {
```

Comment:
Why are most tests in this PR wrapped in await act and an async function that does not await anything or return a promise?

Can some or all of these just be:

```
    render(
      <Web3Container>
        <div>Child</div>
      </Web3Container>
    );
```

if not when do we need this any

Reply:
Because of lazy loaded ag-grid